src/generate.jl

```diff
@@ -57,15 +57,15 @@ emit!(rng, g::TableGenerator, deps, state::Nothing) = emit!(rng, g, deps)
 
 
 """
-    generate([rng::AbstractRNG=GLOBAL_RNG], dag; size::Integer=10) -> Channel{Any}
+    generate([rng::AbstractRNG=GLOBAL_RNG], dag; size::Integer=10) -> Channel{Pair{<:Symbol, <:NamedTuple}}
```

Comment:
I'm not sure about the `<:NamedTuple`, what about Legolas records? Maybe just
```suggestion
    generate([rng::AbstractRNG=GLOBAL_RNG], dag; size::Integer=10) -> Channel{Pair{Symbol, <:Any}}
```
(also we shouldn't need `<:` for Symbol, since it is a concrete type)